### PR TITLE
#265 #75 creating theme and fixing the error icon color

### LINKED ISSUE #265: Incase of error view use Red exclamation
Blue exclamation point icon used instead of the red exclamation point icon when alerting user. Use Red exclamation instead.

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #265, #75

1. This issue took a first pass at creating a theme and  getting all the generalized styling at one place. We could add more stuff to theme but this is a bare bone implementations
2. So far error, warn, success and info to users are coming from theme

This issue touches all of the features. Based on the Feedback i can add more here or later.

### CODE REVIEW

File: ccm_web/client/src/index.tsx [hunk 1]
```diff
@@ -2,17 +2,21 @@ import { SnackbarProvider } from 'notistack'
...
 
 ReactDOM.render(
   <React.StrictMode>
+   <ThemeProvider theme={ccmTheme}>
     <SnackbarProvider maxSnack={3}>
       <BrowserRouter>
         <App />
       </BrowserRouter>
     </SnackbarProvider>
+    </ThemeProvider>
```
Comment (ssciolla) [inline ccm_web/client/src/index.tsx:19]: Looks like the `</ThemeProvider>` closing tag is indented once too far. Unfortunately this doesn't show up on linting, not sure why yet, some JSX rules I think aren't working.

File: ccm_web/client/src/theme.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { createTheme } from '@material-ui/core/styles'
+
+const ccmTheme = createTheme({
```
Comment (ssciolla) [inline ccm_web/client/src/theme.tsx:3]: I think this file doesn't need to be a `.tsx`. There's no JSX here, `.ts` should be fine.

**Review by ssciolla — APPROVED** [review]
This looks good! Couple minor comments for you to consider.

I will note that I did a search for "color: " in `client` and there were a number of other hex values, mostly grayscale stuff I think. It would be nice to tidy this up at some point, but it doesn't have to be as part of this issue.

**Comment by ssciolla** [comment]
Also, this doesn't introduce a `primary` color, which we may want to do before we release.

**Comment by pushyamig** [comment]
> Also, this doesn't introduce a primary color, which we may want to do before we release.

I think is fairly simple to do, But what should be that color? Can we do what Myla is doing `#00274C`

**Comment by ssciolla** [comment]
> > Also, this doesn't introduce a primary color, which we may want to do before we release.
> 
> I think is fairly simple to do, But what should be that color? Can we do what Myla is doing `#00274C`

Mmm, maybe check with Janel, but that's probably reasonable. It doesn't need to be done in this PR.

File: ccm_web/client/src/theme.tsx (hunk 2 above)
Comment (pushyamig) [reply ccm_web/client/src/theme.tsx]: Renamed to `.ts`

File: ccm_web/client/src/index.tsx (hunk 1 above)
Comment (pushyamig) [reply ccm_web/client/src/index.tsx]: I couldn't catch that. Hopefully I fixed it now :)

File: ccm_web/client/src/theme.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { createTheme } from '@material-ui/core/styles'
+
+const ccmTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#00274C'
```
Comment (pushyamig) [inline ccm_web/client/src/theme.ts:6]: I have added Primary color based on Janel suggestion.

**Review by ssciolla — APPROVED** [review]
LGTM!

**Comment by zqian** [comment]
Test successful: 

After I changed the palette colors in theme.ts file (primary, error, warning, success, info), I can verify consistent color changes in all CCM subtask screens.

**Pull request merged**